utils/rpms/daos.spec

```diff
@@ -411,8 +422,15 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 %{_libdir}/libdaos.so
 %{_libdir}/*.a
 
+%files firmware
+# set daos_firmware to be setuid root in order to perform privileged tasks
+%attr(4750,root,daos_server) %{_bindir}/daos_firmware
```

Comment:
So can this be run stand-alone or should/will it always be run from `daos_server`?

Reply:
It's always run by daos_server. We actually restrict who is allowed to call it, similar to daos_admin.

Reply:
Since `daos_server` is already suid-root, does this tool also need to be suid-root or can `daos_server` use it's own root elevation to run this tool with it's own elevated privileges?

Reply:
`daos_server` isn't suid root, is it? I thought that was the reason we were using the privileged binaries like `daos_admin` (and daos_firmware) to begin with.

Reply:
Oh, yes, you are right.  I was mixing them up.